src/citrine/_rest/predictor_metadata.py

```diff
@@ -0,0 +1,71 @@
+from deprecation import deprecated
+from typing import List
+
+from citrine._serialization import properties
+
+
+class PredictorMetadata:
```

Comment:
It's mimicking the structure of the previous classes `AIResource` and `AIMetatdata`, which I'm pretty sure are also always used together, so I assumed there was some reason. I can merge these two into one, sure. But I'm going to keep them separate from `Predictor`, as the combined class will be used by future refactor work we have in the pipeline. If you'd like the name to reflect that now instead of later, just gimme a suggestion and I'd be happy to consider it.